eng/Versions.props

```
@@ -179,7 +179,7 @@
     <!-- ICU -->
     <MicrosoftNETCoreRuntimeICUTransportVersion>7.0.0-rtm.24115.1</MicrosoftNETCoreRuntimeICUTransportVersion>
     <!-- MsQuic -->
-    <MicrosoftNativeQuicMsQuicVersion>2.2.3</MicrosoftNativeQuicMsQuicVersion>
+    <MicrosoftNativeQuicMsQuicVersion>2.2.5-ci.444313</MicrosoftNativeQuicMsQuicVersion>
```

@dotnet/ncl this package version came from internal but it does not exist in public. I see this version was pushed 20 mins ago: https://www.nuget.org/packages/Microsoft.Native.Quic.MsQuic.Schannel/2.3.5 is that the one we need to use?

---

Ah, no, that one is "Schannel". It's a different one.

---

That's the version we need: https://github.com/dotnet/runtime/blob/release/7.0/src/libraries/System.Net.Quic/src/System.Net.Quic.csproj#L107

```suggestion
    <MicrosoftNativeQuicMsQuicVersion>2.3.5</MicrosoftNativeQuicMsQuicVersion>
```